test/unit/mods.example.redmod.autoconversion.ts

```diff
@@ -102,6 +119,31 @@ const ArchiveModToREDmodMigrationSucceeds = new Map<string, ExampleSucceedingMod
       infoNotificationId: InfoNotification.REDmodArchiveAutoconverted,
     },
   ],
+  [
+    `Canonical with single archive migrated to REDmod without name tagging`,
+    {
+      features: FLAG_ENABLED_REDMOD_AUTOCONVERT_ARCHIVES_WITHOUT_TAGGING,
+      expectedInstallerType: InstallerType.Archive,
+      inFiles: [
+        ...ARCHIVE_PREFIXES,
+        path.join(`${ARCHIVE_PREFIX}/first.archive`),
+      ],
+      outInstructions: [
+        movedFromTo(
+          path.join(`${ARCHIVE_PREFIX}/first.archive`),
+          path.join(`${REDMOD_BASEDIR}\\${AUTOCONVERT_MOD_NAME_UNTAGGED}\\${REDMOD_ARCHIVES_DIRNAME}\\first.archive`),
+        ),
+        generatedFile(
+          REDMOD_FAKE_INFO_UNTAGGED_JSON,
+          path.join(`${REDMOD_BASEDIR}\\${AUTOCONVERT_MOD_NAME_UNTAGGED}\\${REDMOD_INFO_FILENAME}`),
+        ),
+        createdDirectory(REDMOD_SCRIPTS_MODDED_DIR),
```

Comment:
what is this created for? or is this just an odd name for the modded directory?

Reply:
I had to go look. Yeah, `export const REDMOD_SCRIPTS_MODDED_DIR = path.join(`r6\\cache\\modded`);` is where the scripts updates go. But also tweaks, I guess, so it's not too accurate.

Reply:
okay then lol